runtime/templates/vcu118/isp_utils.c

```diff
@@ -32,7 +32,13 @@ void isp_test_device_fail(void)
 uint64_t isp_get_cycle_count(uint32_t *result_hi, uint32_t *result_lo)
 {
 #if __riscv_xlen == 64
-	return read_csr(mcycle);
+	uint64_t cycle;
```

Comment:
What's the reason for this change?  Doesn't read_csr already do this?

Reply:
`mcycle` is undefined in this file and the headers it includes.